test/testsuites/watchtests.go

```diff
@@ -25,35 +25,11 @@ const (
 // WatchTests is a test suite that ensure that the watches for child resources
 // are firing correctly and the child resources are restored upon deletion.
 func WatchTests(t *testing.T) {
-	t.Run("restore-csc-catalogsource", testRestoreCscCs)
-	t.Run("restore-csc-deployment", testRestoreCscDeployment)
-	t.Run("restore-csc-service", testRestoreCscService)
 	t.Run("restore-opsrc-catalogsource", testRestoreOpSrcCs)
 	t.Run("restore-opsrc-deployment", testRestoreOpSrcDeployment)
 	t.Run("restore-opsrc-service", testRestoreOpSrcService)
 }
 
-// testRestoreCscCs tests that when a CatalogSource that is owned by a CatalogSourceConfig
-// is restored upon deletion.
-func testRestoreCscCs(t *testing.T) {
-	err := deleteCheckRestoreChild(t, olm.CatalogSourceKind, v2.CatalogSourceConfigKind)
-	assert.NoError(t, err, cscErrMsg)
-}
-
-// testRestoreCscDeployment tests that when a Deployment that is owned by a CatalogSourceConfig
-// is restored upon deletion.
-func testRestoreCscDeployment(t *testing.T) {
-	err := deleteCheckRestoreChild(t, "Deployment", v2.CatalogSourceConfigKind)
-	assert.NoError(t, err, cscErrMsg)
-}
-
-// testRestoreCscService tests that when a Service that is owned by a CatalogSourceConfig
-// is restored upon deletion.
-func testRestoreCscService(t *testing.T) {
-	err := deleteCheckRestoreChild(t, "Service", v2.CatalogSourceConfigKind)
-	assert.NoError(t, err, cscErrMsg)
-}
```

Comment:
Not blocking, just had a comment about the cache sync that exists between the two controllers -- I think we can just rip out any reference to the cache. But if that is a non trivial amount of work, then let's at least update the comments to reflect that more accurately